### PR TITLE
add tests for `remove_if_exists()` and `save_if_exists()`

### PR DESCRIPTION
**Pull request by cjyetman**

### CODE REVIEW

File: tests/testthat/test-save_if_exists.R [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  save_name <- "test.rds"
+
+  save_if_exists(
+    df = df,
+    portfolio_name = portfolio_name,
+    save_name = save_name
+  )
+
+  expect_false(file.exists(save_name))
+})
```
Comment (jdhoffa) [inline tests/testthat/test-save_if_exists.R:40]: Do/ should we expect a warning in this case?

**Review by jdhoffa — APPROVED** [review]

File: tests/testthat/test-save_if_exists.R (hunk 1 above)
Comment (cjyetman) [reply tests/testthat/test-save_if_exists.R:40]: as it is, no... the function does not throw a warning

Comment (jdhoffa) [reply tests/testthat/test-save_if_exists.R:40]: Seems like something we might want to warn about? not a priority though I suppose

Comment (jdhoffa) [reply tests/testthat/test-save_if_exists.R:40]: Captured in #37

**Pull request merged**